src/elements/ngm-layer-legend-container.js

```diff
@@ -0,0 +1,24 @@
+import {LitElement} from 'lit-element';
+import './ngm-layer-legend.js';
+
+class NgmLayerLegendContainer extends LitElement {
+
+  showLegend(config) {
+    console.assert(config.layer);
+    const legendId = `legend_for_${config.layer}`;
+    if (!this.querySelector('#' + legendId)) {
+      const element = document.createElement('ngm-layer-legend');
+      element.id = legendId;
+      element.config = config;
+      element.addEventListener('close', event => event.target.remove());
+      this.appendChild(element);
```

Comment:
It looks strange to do DOM manipulations manually. I would have rather expected the component to maintain a list of "element configs", rendered by lit in the render function.

Reply:
you're absolutely right; what do you think of 9e9c41b435691926dbe5c9c676d68ed8204ea408 ?